README.md

```diff
@@ -1 +1,32 @@
-# beyond09-group01
\ No newline at end of file
+# MeetBalls
+
+Welcome to our new generation social network!
+'MeetBalls' is a web application that is supposed to help people to find partners,

```

Comment:
```suggestion
'MeetBalls' is a web application that is aimed at helping people finding partners,
```
When saying `supposed to`, we often imply that it wasn't actually fulfilled.

Reply:
Thanks, changed